packages/@glimmer/debug-util/test/debug-to-string-test.ts

```diff
@@ -55,6 +55,7 @@ if (import.meta.env.DEV) {
     );
   });
   QUnit.test('should return debug name for class', (assert) => {
+    // eslint-disable-next-line @typescript-eslint/no-extraneous-class
```

Comment:
I don't know how much we care about this lint, but I disabled it for now rather than make too many
changes to the default strict settings in `@typescript-eslint`.